pkgs/unified_analytics/lib/src/survey_handler.dart

```diff
@@ -132,6 +135,11 @@ class Survey {
         samplingRate = json['samplingRate'] is String
             ? double.parse(json['samplingRate'] as String)
             : json['samplingRate'] as double,
+        excludeDashToolList =
+            (json['excludeDashTools'] as List<dynamic>).map((e) {
+          e as String;
+          return DashTool.getDashToolByLabel(e);
+        }).toList(),
```

Comment:
Not that it matters much, but you could shorten the closure a bit by writing is as `(e) => DashTool.getDashToolByLabel(e as String)`. Your choice.

If you choose to return `null` from `getDashToolByLabel` rather than throwing an exception, you'll need to add `.nonNulls` before `.toList`